src/StdCheats.sol

```diff
@@ -416,6 +416,20 @@ abstract contract StdCheatsSafe {
         (addr,) = makeAddrAndKey(name);
     }
 
+    // destroys an account inmediatly, sending the balance to beneficiary 
+    // destroying means: balance will be zero, code will be empty, nonce will be zero
+    // similar to selfdestruct but not identical: selfdestruct destroys code and nonce
+    // only after tx ends, this will run inmediatly
```

Comment:
Just some tweaks to make the comment a bit more readable/fix typos, I got confused at first since sometimes a line break was a new sentence, other times it wasn't
```suggestion
    // Destroys an account immediately, sending the balance to beneficiary.
    // Destroying means: balance will be zero, code will be empty, and nonce will be zero.
    // This is similar to selfdestruct but not identical: selfdestruct destroys code and nonce
    // only after tx ends, this will run immediately.
```